agent/agent_test.go

```diff
@@ -1507,9 +1513,9 @@ func TestAgent_Service_MaintenanceMode(t *testing.T) {
 }
 
 func TestAgent_Service_Reap(t *testing.T) {
-	t.Parallel()
+	// t.Parallel() // timing test. no parallel
```

Comment:
No, because I want people to know why this test is not run in parallel than the rest of the tests in the sam package. This shouldn't be added by accident again.